source/templates/classic/partials/music.ejs

```diff
@@ -20,12 +20,15 @@
         <% } else { %>
           <% if (plugins.music.tracks.length) { %>
             <div class="tracklist">
-              <% for (const {name = "", artist = "", artwork = ""} of plugins.music.tracks) { %>
+              <% for (const {name = "", artist = "", artwork = "", played_at = ""} of plugins.music.tracks) { %>
                 <div class="track">
-                  <img src="data:image/png;base64,<%= artwork %>" width="32" height="32" alt=""/>
+                  <img src="data:image/png;base64,<%= artwork %>" width="<%= !!played_at.length ? 48 : 32 %>" height="<%= !!played_at.length ? 48 : 32 %>" alt=""/>
                   <div class="infos">
-                    <div class="name"><%= name %></div>
+                    <div class="name"><b><%= name %><b></div>
                     <div class="artist"><%= artist %></div>
+                    <% if (played_at.length) {
+                        <div class="played_at"><%= played_at %></div>
+                    <% } %>
```

Comment:
Do you think you can set this as option like `plugin_music_played_at` or `plugin_music_last_played` (you can see this [commit](https://github.com/lowlighter/metrics/commit/64e8afec0d131198aafeeb13e7ea8eb1304fdd89) on how to add a new input, basically you just need to create a new entry in metadata.yml and retrieve its value [here](https://github.com/lowlighter/metrics/blob/master/source/plugins/music/index.mjs#L39)) so users can configure it?

Sorry for small nitpicks, the rest is good to me 👍

Reply:
Will do!